scripted-sbt-redux/src/main/scala/sbt/scriptedtest/ScriptedTests.scala

```diff
@@ -482,102 +403,62 @@ object ScriptedTests extends ScriptedRunner {
 class ScriptedRunner {
   // This is called by project/Scripted.scala
   // Using java.util.List[File] to encode File => Unit
-  def run(
-      resourceBaseDirectory: File,
-      bufferLog: Boolean,
-      tests: Array[String],
-      bootProperties: File,
-      launchOpts: Array[String],
-      prescripted: java.util.List[File],
-  ): Unit = {
-    val logger = new TestConsoleLogger()
-    val addTestFile = (f: File) => { prescripted.add(f); () }
-    run(resourceBaseDirectory, bufferLog, tests, logger, bootProperties, launchOpts, addTestFile)
-    //new FullLogger(Logger.xlog2Log(log)))
-  }
-
-  // This is called by sbt-scripted 0.13.x and 1.x (see https://github.com/sbt/sbt/issues/3245)
-  def run(
-      resourceBaseDirectory: File,
-      bufferLog: Boolean,
-      tests: Array[String],
-      bootProperties: File,
-      launchOpts: Array[String],
-  ): Unit = {
-    val logger = TestConsoleLogger()
-    val prescripted = ScriptedTests.emptyCallback
-    run(resourceBaseDirectory, bufferLog, tests, logger, bootProperties, launchOpts, prescripted)
-  }
-
-  def run(
-      resourceBaseDirectory: File,
-      bufferLog: Boolean,
-      tests: Array[String],
-      logger: AbstractLogger,
-      bootProperties: File,
-      launchOpts: Array[String],
-      prescripted: File => Unit,
-  ): Unit = {
-    // Force Log4J to not use a thread context classloader otherwise it throws a CCE
-    sys.props(org.apache.logging.log4j.util.LoaderUtil.IGNORE_TCCL_PROPERTY) = "true"
-
-    val runner = new ScriptedTests(resourceBaseDirectory, bufferLog, bootProperties, launchOpts)
-    val sbtVersion = bootProperties.getName.dropWhile(!_.isDigit).dropRight(".jar".length)
-    val accept = isTestCompatible(resourceBaseDirectory, sbtVersion) _
-    val allTests = get(tests, resourceBaseDirectory, accept, logger) flatMap {
-      case ScriptedTest(group, name) =>
-        runner.singleScriptedTest(group, name, prescripted, logger)
-    }
-    runAll(allTests)
-  }
-
-  def runInParallel(
-      baseDir: File,
-      bufferLog: Boolean,
-      tests: Array[String],
-      bootProps: File,
-      launchOpts: Array[String],
-      prescripted: java.util.List[File],
-  ): Unit = {
-    runInParallel(baseDir, bufferLog, tests, bootProps, launchOpts, prescripted, 1)
-  }
-
   // This is used by sbt-scripted sbt 1.x
```

Comment:
We need the following version for https://github.com/sbt/sbt/blob/2b557851ca168cd6274adafb7db5d9ed32efb5ee/main/src/main/scala/sbt/ScriptedPlugin.scala#L105-L120

```scala
  // This is used by sbt-scripted sbt 1.x
  def runInParallel(
      baseDir: File,
      bufferLog: Boolean,
      tests: Array[String],
      bootProps: File,
      launchOpts: Array[String],
      prescripted: java.util.List[File],
      instances: Int
  ): Unit = ...
```

https://github.com/sbt/sbt/issues/5592

Reply:
I'll send a PR to recover these entry points.